src/components/facets/facet.component.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { Component } from '@angular/core';
```

Comment:
Since this file only tests facet-check-list it should be facet-check-list.component.spec.ts.